src/OrchardCore.Modules/OrchardCore.Contents/Feeds/CommonFeedItemBuilder.cs

```diff
@@ -26,7 +26,7 @@ public async Task PopulateAsync(FeedContext context)
             {
                 var contentItem = feedItem.Item;
                 var contentItemMetadata = await _contentManager.PopulateAspectAsync<ContentItemMetadata>(contentItem);
-                var bodyAspect = await _contentManager.PopulateAspectAsync<BodyAspect>(contentItem);
```

Comment:
I'm not sure if the `BodyAspect` works well with `MarkdownBodyPart`, because what I had notice is the `bodyAspect` contains the body of the last item all the time?!!

This fix probably not efficient because it requires `MarkdownBodyPart` on the `BlogPost` content type

@deanmarcussen any idea?